apps/condo/domains/miniapp/schema/B2BAppPermission.js

```diff
@@ -19,6 +23,11 @@ const ERRORS = {
         type: PERMISSION_KEY_WRONG_FORMAT_ERROR,
         message: 'Incorrect key format. The key must start with the prefix "can", have lowerCamelCase and answer the question: "what is allowed to the user with this key?". Example: canManagePasses, canReadConfig, etc.',
```

Comment:
There is a `canManagePasses` field for example values.
ADR: https://confluence.in.doma.ai/pages/viewpage.action?pageId=967573505

```suggestion
        message: 'Incorrect key format. The key must start with the prefix "can", have lowerCamelCase and answer the question: "what is allowed to the user with this key?"',
        correctExample: 'canManagePasses'
```

Reply:
`correctExample` is too strange. I have not seen it in any API I've worked with